arangod/RocksDBEngine/RocksDBRecoveryManager.cpp

```diff
@@ -97,9 +99,13 @@ void RocksDBRecoveryManager::start() {
 /// parse recent RocksDB WAL entries and notify the
 /// DatabaseFeature about the successful recovery
 void RocksDBRecoveryManager::runRecovery() {
-  bool success = parseRocksWAL();
-  if (!success) {
-  }  // TODO what do we do if not successful?
+  auto res = parseRocksWAL();
+  if (res.fail()) {
+    LOG_TOPIC(FATAL, Logger::ENGINES)
+      << "Failed during rocksdb WAL recovery - "
+      << res.errorNumber() << " - " << res.errorMessage();
```

Comment:
we normally do not put the errorNumber into the logs in case we have a log message as well.
Using just errorMessage should be sufficient, i.e.
```
+    LOG_TOPIC(FATAL, Logger::ENGINES) << "Failed during RocksDB WAL recovery: " << res.errorMessage();
```